src/libraries/pkg/baseline/packageIndex.json

```diff
@@ -171,7 +173,8 @@
       ],
```

Comment:
Technically we should be setting baselineVersion for all packages.  However it's likely less important here since we don't ever use PackageReferences to the Microsoft.Extensions packages and none of their versions are frozen (so all assembly refs will be to 6.0.0.0 and thus get the 6.0.0-* packages).